scriptlets/scenario

```diff
@@ -24,4 +29,23 @@ dot_to_brackets() {
     '
 }
 
-jq -r -e $(dot_to_brackets "$1") "$SCENARIO_FILE"
+if [ "$1" = "--list" ]; then
+    ARG_LIST_FLAG=true
+    shift
+fi
+
+if [ "$#" -eq 0 ]; then
+    usage
+    echo "Error: missing specifier"
+    exit 1
+fi
+
+RESULT=$(jq -r -e $(dot_to_brackets "$1") "$SCENARIO_FILE")
+EXIT_CODE="$?"
+
+if [ "$EXIT_CODE" -ne 0 ] || [ -z "$ARG_LIST_FLAG" ]; then
+    echo "$RESULT"
+else
+    echo "$RESULT" | jq -r 'join(" ")'
```

Comment:
Is this a footgun waiting to shoot? If we use this in a `$( ... )` then its a space/command injection etc. waiting to happen